morango/__init__.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+default_app_config = 'morango.apps.MorangoConfig'
```

Comment:
yeah our minimal django will be probably be 1.8 or 1.9.
If we put the code there, that will also be called everytime on app load, correct?
If that's the case, then I might prefer having it there.